### ISSUE TITLE
Problem with webpack

### ISSUE THREAD

**Issue by Carlos123211** [verifalia/verifalia-js-sdk#2]
I had installed the sdk using yarn. When I follow the example on the read.me I get this error
```
verifalia__WEBPACK_IMPORTED_MODULE_8__.VerifaliaRestClient is not a constructor
TypeError: verifalia__WEBPACK_IMPORTED_MODULE_8__.VerifaliaRestClient is not a constructor
```
Is it due to I installed the SDK on Angular ? .

**Comment by verifalia** [comment]
Thanks for having reported this issue: we are going to include a patch for this in the upcoming release for this SDK, which is due this week.

As a nasty work-around for this import problem, you can:

- copy the `verifalia.mjs` file in the `node_modules/verifalia/dist/es2015/` folder of your project into `node_modules/verifalia/dist/es2015/verifalia.js` (that is, same folder - just remove the "m" in the extension of the copied file name);
- change your import into the following:
```javascript
import { VerifaliaRestClient } from 'verifalia/dist/es2015/verifalia';
```
That will do the trick. Unfortunately, typings won't work correctly using this work-around, so you won't have code completion on your favorite editor.

**Comment by verifalia** [comment]
This is now fixed, as of v3.0. Please see the updated README.md for instructions and examples about how to import and use the library in your Angular project.
